bundles/binding/org.openhab.binding.fritzbox/README.md

```diff
@@ -10,6 +10,15 @@ This binding can be configured in the `services/fritzbox.cfg` file.
 | password |         |   Yes    | Password to your Fritz!Box |
 | user     |         | when user management is enabled | User of your Fritz!Box. It is a good practice to create an additional user for openHAB. |
 
+
+As a result, your config might look like the following:
```

Comment:
Probably better to leave these examples out, since they're incorrect for the current version of OH.

Reply:
If the example is wrong, then please modify it and explain it how it is done better?

Because the example was missing I had to dig Google to find https://github.com/openhab/openhab1-addons/wiki/fritz-box-binding to get which syntax the binding is using.

Reply:
The correct way in OH2 is

````
ip=<ipaddress>
password=<password>
user=<username>
````